src/server/scripts/Commands/cs_reload.cpp

```diff
@@ -436,81 +436,76 @@ class reload_commandscript : public CommandScript
+            cInfo->trainer_spell            = fields[32].GetUInt32();
+            cInfo->trainer_class            = fields[33].GetUInt8();
+            cInfo->trainer_race             = fields[34].GetUInt8();
+            cInfo->type                     = fields[35].GetUInt8();
+            cInfo->type_flags               = fields[36].GetUInt32();
+            cInfo->lootid                   = fields[37].GetUInt32();
+            cInfo->pickpocketLootId         = fields[38].GetUInt32();
+            cInfo->SkinLootId               = fields[39].GetUInt32();
 
             for (uint8 i = SPELL_SCHOOL_HOLY; i < MAX_SPELL_SCHOOL; ++i)
-                cInfo->resistance[i] = fields[46 + i - 1].GetUInt16();
-
-            cInfo->spells[0]          = fields[52].GetUInt32();
-            cInfo->spells[1]          = fields[53].GetUInt32();
-            cInfo->spells[2]          = fields[54].GetUInt32();
-            cInfo->spells[3]          = fields[55].GetUInt32();
-            cInfo->spells[4]          = fields[56].GetUInt32();
-            cInfo->spells[5]          = fields[57].GetUInt32();
-            cInfo->spells[6]          = fields[58].GetUInt32();
-            cInfo->spells[7]          = fields[59].GetUInt32();
-            cInfo->PetSpellDataId     = fields[60].GetUInt32();
-            cInfo->VehicleId          = fields[61].GetUInt32();
-            cInfo->mingold            = fields[62].GetUInt32();
-            cInfo->maxgold            = fields[63].GetUInt32();
-            cInfo->AIName             = fields[64].GetString();
-            cInfo->MovementType       = fields[65].GetUInt8();
-            cInfo->InhabitType        = fields[66].GetUInt8();
-            cInfo->HoverHeight        = fields[67].GetFloat();
-            cInfo->ModHealth          = fields[68].GetFloat();
-            cInfo->ModMana            = fields[69].GetFloat();
-            cInfo->ModArmor           = fields[70].GetFloat();
-            cInfo->RacialLeader       = fields[71].GetBool();
-            cInfo->movementId         = fields[72].GetUInt32();
-            cInfo->RegenHealth        = fields[73].GetBool();
-            cInfo->MechanicImmuneMask = fields[74].GetUInt32();
-            cInfo->flags_extra        = fields[75].GetUInt32();
-            cInfo->ScriptID           = sObjectMgr->GetScriptId(fields[76].GetCString());
+                cInfo->resistance[i] = fields[40 + i - 1].GetUInt16();
+
+            cInfo->spells[0]                = fields[46].GetUInt32();
+            cInfo->spells[1]                = fields[47].GetUInt32();
+            cInfo->spells[2]                = fields[48].GetUInt32();
+            cInfo->spells[3]                = fields[49].GetUInt32();
+            cInfo->spells[4]                = fields[50].GetUInt32();
+            cInfo->spells[5]                = fields[51].GetUInt32();
+            cInfo->spells[6]                = fields[52].GetUInt32();
+            cInfo->spells[7]                = fields[53].GetUInt32();
```

Comment:
This can be done here if you have time
```
            for (uint8 i = 0; i < CREATURE_MAX_SPELLS; ++i)
                cInfo->spells[i] = fields[46 + i].GetUInt32();
```

Reply:
Sure will add that